docs/RN/Kubernetes-Operator-for-PSMONGODB-RN1.17.0.md

```diff
@@ -73,6 +73,8 @@ Finalizers were renamed to contain fully qualified domain names to comply with t
 * `PerconaServerMongoDBBackup` Custom Resource:
     * `delete-backup` finalizer renamed to `percona.com/delete-backup`
 
+Also, the [`psmdb-db` Helm chart](https://github.com/percona/percona-helm-charts/tree/main/charts/psmdb-db) undergoes rename of one parameter: starting from this release, the old parameter `users` that allows specifying [system users](../users.md#system-users) is renamed to `systemUsers`. Now parameter `users` is referring to the new [Declarative user management](../users.md#create-users-in-the-custom-resource).
```

Comment:
How about simplifying it a bit? + I think it is worth mentioning that it impacts users who are going to upgrade.

Key change in [`psmdb-db` Helm chart](https://github.com/percona/percona-helm-charts/tree/main/charts/psmdb-db): the parameter for defining [system users](../users.md#system-users) has been renamed from `users` to systemUsers. The users parameter now handles the new [Declarative user management](../users.md#create-users-in-the-custom-resource) feature. This change impacts users upgrading to this version, make sure that values manifests are changed accordingly.